suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -95,11 +95,3 @@ tests:
       name: Check trash if the deleted images are moved to trash automatically
       polarion-id: CEPH-83573297
 
-  - test:
-      config:
-        branch: master
-        script_path: qa/workunits/rbd
-        script: krbd_rxbounce.sh
-      desc: Executig upstream RBD kernel exclusive scenarios
-      module: test_rbd.py
-      name: 8_rbd_krbd_rxbounce
```

Comment:
Removed this embedded test which was added by mishap as part of PR #1664
The test was used only for kernel fix test.

cc @preethisn